src/main/scala/codacy/checkstyle/Checkstyle.scala

```diff
@@ -42,6 +44,11 @@ object Checkstyle extends Tool {
     run(filesToLint, config)
   }
 
+  private def setCheckstyleConfigLocationProperty(configFilePath: String) = {
+    val parentPath = File(configFilePath).path.getParent.toAbsolutePath.toString
```

Comment:
You can use the better files facilities.
```suggestion
    val parentPath = File(configFilePath).parent.pathAsString
```